__tests__/app.test.js

```diff
@@ -47,4 +47,32 @@ describe('GET /api/topics', () => {
         })
     })
     });
-// test push
\ No newline at end of file
+// test push
+describe('GET/api/articles/:article_id', () => {
+  it('GET:200 and should respond with an object containing correct article properties', () => {
+    return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then((response) => {
+        const { article } = response.body;
+        expect(typeof article).toBe('object')
+        expect (article.hasOwnProperty('author')).toBe(true);
+        expect (article.hasOwnProperty('title')).toBe(true);
+        expect (article.hasOwnProperty('article_id')).toBe(true);
```

Comment:
You can be abit more specific here - what number do you expect the article id to be?